documentation/content/en/books/handbook/cutting-edge/_index.adoc

```diff
@@ -259,7 +259,7 @@ Both types of upgrade can be performed by providing `freebsd-update` with a rele
 
 [WARNING]
 ====
-After each new `RELEASE`, FreeBSD package build servers will, for a limited period, *not* use the newest version of the operating system.
+After each new `RELEASE`, FreeBSD package build servers will, for a limited period, *not* use the newer version of the operating system.
```

Comment:
For future reference: 

* this first sentence is not accurate – not _entirely_ true. 

If I had made the paragraph accurate, then the (relative) complexity would have detracted from the key point of the warning. 

----

Generally, I try to avoid warning boxes and note boxes – their presence sometimes signals an underlying need to restructure things. 

In this case, I chose a warning box because we were approaching the date of release of 13.2. 

(It's easier/quicker to gain approval to commit a condensed warning, than it is to gain approval for a broader restructure.)